library/modules/bigip_profile_client_ssl.py

```diff
@@ -601,10 +601,10 @@ def cert_key_chain(self):
                 'key': fq_name(self.partition, key),
                 'chain': chain
             }
-            if 'passphrase' in item:
+            if 'passphrase' in item and item['passphrase'] not in ('None', None, 'none'):
                 tmp['passphrase'] = item['passphrase']
-            if 'true_names' in item:
-                tmp['true_names'] = item['true_names']
+            # if 'true_names' in item:
+            #     tmp['true_names'] = item['true_names']
```

Comment:
if these are not needed then its fine to remove it, as long as true_names work without it

Reply:
Yeah, it works without this. The idea is that true_names should be used to make the logic decision of whether to append the filenames. But we don't pass it to the REST API. So there's no reason to add it to the tmp variable. 

For that matter, the idempotency actually fails because when it compares want with have, it always says that want includes true_names, which have doesn't contain. This is clearly because no testing was written for this newly added parameter.